templatefuncs.go

```diff
@@ -207,6 +279,35 @@ func toStringTemplateFunc(arg any) string {
 	}
 }
 
+// uniqTemplateFunc is the core implementation of the `uniq` template function.
+func uniqTemplateFunc(list []any) []any {
+	seen := make(map[any]struct{})
```

Comment:
This will not work as expected. For example, see https://go.dev/play/p/qemCBLcOyPh. I think it can fail in other ways too.